includes/class-wc-customizer-settings.php

```diff
@@ -431,6 +431,20 @@ public function get_settings() {
 
 					array( 'type' => 'sectionend' ),
 
+					array(
+						'title' => __( 'Cart Button', 'woocommerce-customizer' ),
+						'type'  => 'title'
+					),
+
+					array(
+						'id'       => 'woocommerce_return_to_shop_button',
+						'title'    => __( 'Empty Cart', 'woocommerce-customizer' ),
+						'desc_tip' => __( 'Change the cart button text for empty cart page', 'woocommerce-customizer' ),
```

Comment:
How about `Changes the Return to Shop button text on cart`? 

To consistent with others description.